semgrep-core/src/core/Pattern_match.ml

```diff
@@ -101,6 +101,8 @@ and rule_id = {
   id : string;
   (* other parts of a rule (or mini_rule) used in JSON_report.ml *)
   message : string;
+  fix : string option;
```

Comment:
maybe add a TODO comment saying "maybe we should remove those fields and just have a rule: Rule.t;"